src/react-queries/index.ts

```diff
@@ -134,63 +138,58 @@ const mutations = createMutationKeys('mutations', {
     },
   }),
 })
+
+type Period = 'all'
+type Interval = '5min' | 'hour' | 'day' | 'week' | 'month' | 'quarter' | 'year'
+
 // Feature-agnostic, abstracts away midgard endpoints
 const midgard = createQueryKeys('midgard', {
-  swapsData: (assetId: AssetId | undefined, timeframe: '24h' | 'previous24h' | '7d') => ({
-    queryKey: ['midgardSwapsData', assetId ?? '', timeframe],
+  swapsData: (poolAssetId: string, interval: Interval, count: number) => ({
```

Comment:
Note, `count` will invalidate queries, we may want to take a look at [pagination](https://tanstack.com/query/v4/docs/framework/react/guides/paginated-queries) in a follow-up when implementing charts